### PR TITLE
5.22.0 changelog

### PR DESCRIPTION
**Pull request by hao-nan-li**

### CODE REVIEW

**Comment by hao-nan-li** [comment]
For clarification purpose: Added shuya as reviewer to verify the `breaking-change` release note.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+
+FEATURES:
+* **New Data Source:** `google_cloud_quotas_quota_infos` ([#7116](https://github.com/hashicorp/terraform-provider-google-beta/pull/7116))
+* **New Resource:** `google_access_context_manager_service_perimeter_dry_run_resource` ([#7115](https://github.com/hashicorp/terraform-provider-google-beta/pull/7115))
+* **New Resource:** google_cloud_quotas_quota_info ([#7092](https://github.com/hashicorp/terraform-provider-google-beta/pull/7092))
+
+IMPROVEMENTS:
+* accesscontextmanager: supported managing service perimeter dry run resources outside the perimeter via new resource `google_access_context_manager_service_perimeter_dry_run_resource` ([#7115](https://github.com/hashicorp/terraform-provider-google-beta/pull/7115))
+* cloudrunv2: added the validation to restrict number of ports to 1 to fail earlier than server-side validation ([#7103](https://github.com/hashicorp/terraform-provider-google-beta/pull/7103))
```
Comment (c2thorn) [inline CHANGELOG.md:13]:
```suggestion
* cloudrunv2: added plan-time validation to restrict number of ports to 1 in `google_cloud_run_v2_service` ([#7103](https://github.com/hashicorp/terraform-provider-google-beta/pull/7103))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+* cloudrunv2: supported mounting Cloud Storage buckets using GCSFuse in `google_cloud_run_v2_job` (beta). ([#7102](https://github.com/hashicorp/terraform-provider-google-beta/pull/7102))
```
Comment (c2thorn) [inline CHANGELOG.md:14]:
```suggestion
* cloudrunv2: supported mounting Cloud Storage buckets using GCSFuse in `google_cloud_run_v2_job` ([#7102](https://github.com/hashicorp/terraform-provider-google-beta/pull/7102))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+* composer: small fixes and enhancements after the bugbash. ([#7120](https://github.com/hashicorp/terraform-provider-google-beta/pull/7120))
```
Comment (c2thorn) [inline CHANGELOG.md:15]: This just needs to be rewritten completely

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+* compute: Added enumeration value `SEV_LIVE_MIGRATABLE_V2` for the `guest_os_features` of `google_compute_disk` ([#7123](https://github.com/hashicorp/terraform-provider-google-beta/pull/7123))
```
Comment (c2thorn) [inline CHANGELOG.md:16]:
```suggestion
* compute: added enumeration value `SEV_LIVE_MIGRATABLE_V2` for the `guest_os_features` of `google_compute_disk` ([#7123](https://github.com/hashicorp/terraform-provider-google-beta/pull/7123))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
...
+* compute: added `status.all_instances_config.revision` field to `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager` ([#7104](https://github.com/hashicorp/terraform-provider-google-beta/pull/7104))
+* compute: added field `path_template_match` to resource `google_compute_region_url_map` ([#7094](https://github.com/hashicorp/terraform-provider-google-beta/pull/7094))
+* compute: added field `path_template_rewrite` to resource `google_compute_region_url_map` ([#7094](https://github.com/hashicorp/terraform-provider-google-beta/pull/7094))
+* pubsub: added `ingestion_data_source_settings` field to `google_pubsub_topic` resource ([#7111](https://github.com/hashicorp/terraform-provider-google-beta/pull/7111))
+* storage: added 'soft_delete_policy' to 'google_storage_bucket' resource ([#7119](https://github.com/hashicorp/terraform-provider-google-beta/pull/7119))
+* workstations: added `host.gceInstance.boostConfig` to `google_workstations_workstation_config` (beta) ([#7122](https://github.com/hashicorp/terraform-provider-google-beta/pull/7122))
```
Comment (c2thorn) [inline CHANGELOG.md:22]:
```suggestion
* workstations: added `host.gceInstance.boostConfig` to `google_workstations_workstation_config` ([#7122](https://github.com/hashicorp/terraform-provider-google-beta/pull/7122))
```

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (hao-nan-li) [reply CHANGELOG.md]: Attempted to re-write this entry.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,31 @@
...
+* **New Resource:** `google_access_context_manager_service_perimeter_dry_run_resource` ([#7115](https://github.com/hashicorp/terraform-provider-google-beta/pull/7115))
+* **New Resource:** `google_cloud_quotas_quota_info` ([#7092](https://github.com/hashicorp/terraform-provider-google-beta/pull/7092))
+
+IMPROVEMENTS:
+* accesscontextmanager: supported managing service perimeter dry run resources outside the perimeter via new resource `google_access_context_manager_service_perimeter_dry_run_resource` ([#7115](https://github.com/hashicorp/terraform-provider-google-beta/pull/7115))
+* cloudrunv2: added plan-time validation to restrict number of ports to 1 in `google_cloud_run_v2_service` ([#7103](https://github.com/hashicorp/terraform-provider-google-beta/pull/7103))
+* cloudrunv2: supported mounting Cloud Storage buckets using GCSFuse in `google_cloud_run_v2_job` ([#7102](https://github.com/hashicorp/terraform-provider-google-beta/pull/7102))
+* composer: added field `count` to validate number of DAG processors. ([#7120](https://github.com/hashicorp/terraform-provider-google-beta/pull/7120))
```
Comment (c2thorn) [inline CHANGELOG.md:15]:
```suggestion
* composer: added field `count` to validate number of DAG processors in `google_composer_environment` ([#7120](https://github.com/hashicorp/terraform-provider-google-beta/pull/7120))
```

**Review by c2thorn — APPROVED** [review]

**Pull request merged**